src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -200,7 +200,7 @@ public Category GetCategory
         /// Returns the FamilyType for this Element. Returns null if the Element cannot have a FamilyType assigned.
```

Comment:
should the comment say `ElementType` instead of `FamilyType` ?

Reply:
hmm good question, didnt really change this only edited the return type to ElementType instead of Element.